### PR TITLE
Create themessaging.co.xml

### PR DESCRIPTION
**Pull request by dntfeedthemnkys**
This autoconfig file contains an updated list of some of the domains managed by The Messaging Company. 
For more information on The Messaging Company, please visit: https://themessaging.co

### CODE REVIEW

**Review by cketti — COMMENTED** [review]
@dntfeedthemnkys: Autoconfig is meant to be a decentralized protocol. We'd much prefer that email providers hosted their config under `https://autoconfig.{emaildomain}/mail/config-v1.1.xml`. I'd be happy to talk you through that if you're interested (email: cketti@thunderbird.net).

**Comment by nathansalt** [comment]
> @dntfeedthemnkys: Autoconfig is meant to be a decentralized protocol. We'd much prefer that email providers hosted their config under `https://autoconfig.{emaildomain}/mail/config-v1.1.xml`. I'd be happy to talk you through that if you're interested (email: [cketti@thunderbird.net](mailto:cketti@thunderbird.net)).

Thanks @cketti ! Looks like our hosted xml generator works with that URL...just had to make an nginx change.
Someone had configured it to autoconfig.{emaildomain}/autoconfig/mail/config-v1.1.xml which obviously doesn't work.

@dntfeedthemnkys I think you can cancel this PR.

File: ispdb/themessaging.co.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<clientConfig version="1.1">
+  <emailProvider id="themessaging.co">
+    <domain>themessaging.co</domain>
```
[the remark this answers is not in the document]
Comment (dntfeedthemnkys) [reply ispdb/themessaging.co.xml:4]: Thanks @cketti - brain burp here, should have had themessagingco.com.au.  Regardless, following on from Salty's comments below, I'll cancel this out for now and we'll retest.

**Comment by dntfeedthemnkys** [comment]
Issue with our nginx configuration. Closing this and retesting.

**Pull request closed without merging**